scalding-thrift-macros-fixtures/src/main/scala/com/twitter/scalding/thrift/macros/scalathrift/TestEnum.scala

```diff
@@ -0,0 +1,94 @@
+/**
+ * Generated by Scrooge
+ *   version: ?
+ *   rev: ?
+ *   built at: ?
+ */
+package com.twitter.scalding.thrift.macros.scalathrift
```

Comment:
I'd rather just use the scrooge plugin and generate these, personally (also it is a better test, because scrooge will evolve).